documentation/book/master.adoc

```diff
@@ -37,9 +37,8 @@ ifdef::InstallationAppendix[]
 include::appendix_deploying_kubernetes_openshift_cluster.adoc[]
 endif::InstallationAppendix[]
 
-ifdef::MetricsAppendix[]
-include::appendix_metrics.adoc[]
-endif::MetricsAppendix[]
+[appendix]
+include::assembly-metrics-setup.adoc[leveloffset=+1]
```

Comment:
Why do we don't need the `ifdef` anymore?

Reply:
Assuming this will be pulled downstream when ready.